src/plugins/legacy/itkDataImage/readers/itkDataImageReaderCommand.h

```diff
@@ -15,30 +16,36 @@
 
 class dtkAbstractDataReader;
 
-namespace itk {
-
-  class DataImageReaderCommand: public Command {
-  public:
+namespace itk
+{
+class DataImageReaderCommand: public Command
+{
+public:
 
     typedef DataImageReaderCommand         Self;
     typedef Command                        Superclass;
     typedef itk::SmartPointer<Self>        Pointer;
     typedef itk::SmartPointer<const Self>  ConstPointer;
     
-    itkTypeMacro( DataImageReaderCommand, Command );
-    itkNewMacro (Self);
+    itkTypeMacro( DataImageReaderCommand, Command )
+    itkNewMacro (Self)
     
     void Execute(Object *caller, const EventObject &event);
     void Execute(const Object *caller, const EventObject &event);
     
     void SetDataImageReader (dtkAbstractDataReader* reader)
-    { m_Reader = reader; }
-
-  protected:
-    DataImageReaderCommand(){ m_Reader = 0; };
-    virtual ~DataImageReaderCommand(){};
+    {
+        m_Reader = reader;
+    }
+
+protected:
+    DataImageReaderCommand()
+    {
+        m_Reader = nullptr;
+    }
+    virtual ~DataImageReaderCommand(){}
```

Comment:
i do not change for now the functions to =default ones. I made the mistake last week to change a lot of them and i had compilation errors everywhere. However, this in on my code cleaning list (with a lot of other stuff). I'm going to do a specific PR for this topic :)

Reply:
If I can give my personal opinion on this issue:
I think we should remove these empty destructor definitions instead of replacing them with `~foo() override = default;`. There are very few cases where this explicit declaration of the default destructor is needed so this is a good occasion to reduce the code bloat of medInria. As long as the base classes have virtual destructors, we're good.

Reply:
Maybe we could create an issue with the cleaning process rules : NULL -> nullptr, virtual -> override, private classes to remove, =default, etc. This way, we could list our format rules, and debate if needed also ^^

Reply:
The question here is more about keeping code that is useless. We can standardise it to newer C++ syntax, but it is still useless.

Reply:
(last comment edited to make it clearer)